datasets/huishoudelijkafval/ticket/v1.0.0.json

```diff
@@ -0,0 +1,249 @@
+{
+    "id": "ticket",
+    "version": "1.0.0",
+    "type": "table",
+    "provenance": "afval_api_ticket",
+    "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "mainGeometry": "geometrie",
+        "identifier": "id",
+        "required": [
+            "id",
+            "schema"
+        ],
+        "display": "id",
+        "properties": {
+            "schema": {
+                "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+            },
+            "id": {
+                "type": "integer",
+                "description": "Identificerend kenmerk van het ticket."
+
+            },
+            "ticketNummer": {
+                "type": "integer",
+                "description": "Nummer van de ticket. Een ticket kan over meer dan één container of containerlocatie gaan."
+            },
+            "container": {
+                "type": "integer",
+                "relation": "huishoudelijkafval:container",
+                "provenance": "bedrijfsmiddel_container",
+                "description": "Identificerend kenmerk van de container."
+            },
+            "containerlocatie": {
+                "type": "integer",
+                "relation": "huishoudelijkafval:containerlocatie",
+                "provenance": "bedrijfsmiddel_containerlocatie",
+                "description": "Identificerend kenmerk van de containerlocatie."
+            },
+            "datumCreatie": {
+                "type": "datetime",
+                "description": "De datum waarop het ticket aangemaakt is."
+            },
+            "plaatje": {
+                "type": "array",
```

Comment:
When this is a url, it should be `"type": "string", "format": "uri"`.

Reply:
aangepast